realm/realm-jni/project.mk

```diff
@@ -52,7 +52,7 @@ ifeq ($(REALM_ANDROID),)
     endif
   endif
 else
-  PROJECT_CFLAGS += -fvisibility=hidden -DANDROID
+  PROJECT_CFLAGS += -fvisibility=hidden -DANDROID -ffunction-sections -fdata-sections -flto
```

Comment:
This file is included by the `generic.mk` but `PROJECT_CFLAGS` doesn't seem to be used by `realm-jni` . Is this change from core's change?